support/ruby/collins-client/lib/collins/api/ip_address.rb

```diff
@@ -33,11 +33,12 @@ def ipaddress_update! asset_or_tag, old_address = nil, options = {}
       end
     end
 
-    def ipaddress_delete! asset_or_tag, pool = nil
+    def ipaddress_delete! asset_or_tag, pool = nil, address = nil
```

Comment:
Good idea - I'll refactor that.  My initial intent was to minimize other code changes but I think an options hash makes more sense.